### DISCUSSION TITLE
NEED HELP WITH LICENSE PURCHASE DELIVERY CORRECTION

### DISCUSSION THREAD

**Discussion by neilaspare** [helgoboss/helgobox#1330] [General]
I just purchased a license for playtime, HOWEVER, the email address that my paypal order was/is currently tied to is an email address that I am unable to actually access.  The email address used for the order was a23p@protonmail.com
and apparently that is the email address the license was being sent to.  
However, as that email address is one I can not currently access, the email address I would have liked things sent to is 23.alien@gmail.com

Please let me know what can be done to correct and resolve this issue.

Best regards
-A.

**Comment by helgoboss** [comment]
Hi!

I sent the license to your other email address.

Please use info@helgoboss.org for such inquiries in the future. Whatever you write here is public to everyone! Feel free to delete this discussion.

Cheers
Benjamin

**Comment by neilaspare** [comment]
THank you!

On Mon, Nov 18, 2024 at 5:10 AM Benjamin Klum ***@***.***>
wrote:

> Hi!
>
> I sent the license to your other email address.
>
> Please use ***@***.*** for such inquiries in the future. Whatever
> you write here is public to everyone! Feel free to delete this discussion.
>
> Cheers
> Benjamin
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/helgoboss/helgobox/discussions/1330#discussioncomment-11290162>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BNAXJLFVGLDRIWOKXRXTZUL2BG4J5AVCNFSM6AAAAABR7B5KPGVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCMRZGAYTMMQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
93 93/93

www.twenty3.tk
www.acidallstars.com
www.myspace.com/aleph23
www.myspace.com/a20threep
